pandas/tests/io/excel/test_odswriter.py

```diff
@@ -3,63 +3,63 @@
     datetime,
 )
 import re
+import uuid
 
 import pytest
 
-from pandas.compat import is_platform_windows
-
 import pandas as pd
-import pandas._testing as tm
 
 from pandas.io.excel import ExcelWriter
 
 odf = pytest.importorskip("odf")
 
-if is_platform_windows():
-    pytestmark = pytest.mark.single_cpu
-
 
 @pytest.fixture
 def ext():
     return ".ods"
 
 
-def test_write_append_mode_raises(ext):
+@pytest.fixture
+def tmp_excel(ext, tmp_path):
```

Comment:
Why is this fixture just for excel?

Reply:
I think our Excel code is picky about checking the file extension and hence if/how `ext` is defined in our tests. I think in the general case this won't matter but there might be a better way to generalize how this is set up in our Excel tests?

Reply:
Sounds good. Not a blocker by any means